git_hooks/README.md

```diff
@@ -0,0 +1,14 @@
+README for git hooks script
+===========================
+git has a way to run scripts, which are invoked by specific git commands.
+The git hooks are located in <root>.git/hooks, and as such are not under version control
+for more information, see https://git-scm.com/docs/githooks
+
+The mbed TLS git hooks are located in <root>/git_hooks directory, and one must create a soft link from <root>.git/hooks to <root>/git_hooks, in order to make the hook scripts successfully work.
+example:
+
+on linux:
+.git/hooks> ln -s ../../git_hooks/pre-push pre-push
+on windows, as administrator:
+.git\hooks> mklink pre-push ..\..\git_hooks\pre-push
```

Comment:
If you take a look at this file in the GitHub web interface after the markdown has been stripped and the text formatted, you will find that these four lines of text are merged into a single long "sentence" that is not reasy to read. Please consider using the morkdown syntax [explained here](https://guides.github.com/features/mastering-markdown/). For example, it would be more readable if the text looked like show below:

Run the following command to create a link on linux:
```
ln -s git_hooks/pre-push .git/hooks/pre-push
```
On windows, execute the following command while running as administrator:
```
mklink .git/hooks/pre-push git_hooks\pre-push
```